frontend/services/project-developers/projectDevelopersService.ts

```diff
@@ -0,0 +1,54 @@
+import { useMemo } from 'react';
```

Comment:
Same here, the file could be renamed as `frontend/services/project-developers.ts`.